### ISSUE TITLE
Improvement: Storing gameplaytags within game feature plugin

### ISSUE THREAD

**Issue by braheem** [x157/x157.github.io#3]
I recently figured out that you can store all plugin-specific gameplay tags with the game feature plugin, so you don't have to go into the project settings to add any tags and then worry about how you will backup/restore those tags in future iterations when you update lyra.

If you look at the folder directory (windows explorer not ue5) for ShooterCore or TopDownArena, you'll see that they have a Config/Tags directory where the gameplay tags specific to both of these plugins are stored.

For example in my project you would find the shootercore tags in  `LyraJuly30\Plugins\GameFeatures\ShooterCore\Config\Tags`

You can add the same for XistGame. So `${LyraDirectory}/Plugins/GameFeatures/XistGame/Config/Tags/XistGameTags.ini`, and you can add tags there , example below:

```
[/Script/GameplayTags.GameplayTagsList]
GameplayTagList=(Tag="ShooterGame.GamePhase.Buy",DevComment="")
GameplayTagList=(Tag="GameplayCue.Character.Teleport",DevComment="")
GameplayTagList=(Tag="GameplayCue.Character.Teleport.End",DevComment="")
GameplayTagList=(Tag="GameplayCue.Character.Teleport.Start",DevComment="")
GameplayTagList=(Tag="Ability.Type.Action.Teleport",DevComment="")
GameplayTagList=(Tag="InputTag.Ability.Teleport",DevComment="")
GameplayTagList=(Tag="Event.Movement.Teleport",DevComment="")
```

I believe that if you restart unreal engine it should autopopulate the gameplay tags list in project settings with these tags. If it doesn't  then you can simply add the new file as a tag source: 
![image](https://user-images.githubusercontent.com/4989986/182620371-ddd1947e-23d2-437b-b426-3ee2a3c479ec.png)

Up to you but this can give an alternate instruction in the following step of your guide so that others can better encapsulate their gamefeature plugin content, instead of adding their tags in project settings:
![image](https://user-images.githubusercontent.com/4989986/182623364-ce2e484c-cfa0-4297-8ceb-fdb6eff93856.png)

**Comment by x157** [comment]
I agree it's definitely important to let people know how to encapsulate Gameplay Tags in their project.

I probably won't modify that specific tutorial you mention as that is linked to a YT video that follows the procedure there, and is mainly focused on how to get the Ability working rather than the correct way to add Tags.

However this definitely does seem like a useful topic that people would be interested in:  "What's the RIGHT way to add Gameplay Tags to my project?"

Probably will link it from here:  https://x157.github.io/UE5/LyraStarterGame/

Setting this as an enhancement.  When I get to a stopping point on my current project I'll circle back and add this.  Thanks for the suggestion!
